components/board/genericlinux/board.go

```diff
@@ -293,7 +325,11 @@ func (b *sysfsBoard) AnalogReaderNames() []string {
 }
 
 func (b *sysfsBoard) DigitalInterruptNames() []string {
-	return nil
+	names := []string{}
+	for name := range b.interrupts {
```

Comment:
[minor] As we talked about - if b.interrupts is nil because a periph board does not construct it - catch that and return nil to preserver old behaviour.